css/selectors/highlight.json

```diff
@@ -13,7 +13,9 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "preview",
+              "partial_implementation": true,
+              "notes": "Cannot yet be used with <code>text-decoration</code> and <code>text-shadow</code>. See <a href='https://bugzil.la/1703961'>bug 1703961</a>."
```

Comment:
Note, this is partial and preview, but is  [expected](https://bugzilla.mozilla.org/show_bug.cgi?id=1840818#c12) to match spec in on release later this half. The bugzilla link is to the parent bug. The text refers to missing functionality, that users might expect; there are other things like inheritance that are bugs - those I have not mentioned.